api/src/routers/meals.js

```diff
@@ -0,0 +1,71 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const router = express.Router();
+
+// GET /api/meals - Returns all meals
+router.get("/", async (req, res) => {
+  try {
+    const meals = await knex.select("*").from("meals");
```

Comment:
`select("*")` is the default, we can live without it :)
also `from("meals")` you can move that into  `knex("meals")`, like in the other endpoints